doc/declarations.md

```diff
@@ -528,9 +566,29 @@ by rw [div]; rfl
 end Hide
 ```
 
+If Lean cannot find a permutation of the arguments for which all recursive calls are decreasing, it will print a table that contains, for every recursive call, which arguments Lean could prove to be decreasing.
+
+Lean will print the termination argument it found if ``set_option showInferredTerminationBy true`` is set.
```

Comment:
For the table I wrote it while you posted.

Adding an example for the output from ``showInferredTerminationBy`` here is almost not helpful, because the syntax it is presented in is discussed later, and clearly recognizable as that.

Reply:
Sounds good